Add a clarification on Reonboarding and Router Devices (#140)
DKE-Data/agrirouter-interface-documentation: modules/ROOT/pages/certification.adoc

|xref:./integration/reonboarding.adoc[Re-onboarding]
|Always

> (if no RouterDevice is used)

Reviewer: ```suggestion
(if the application does not use router devices)
```

** Wrong account: During re-onboarding, the user is logged in with a different agrirouter account than before. This should result in a new endpoint onboarding in a different account.

|xref:./router-devices.adoc[Updating RouterDevice]
| Farming Software

Telemetry Platforms

> (If RouterDevice is used)

Reviewer: ```suggestion
(If the application uses router devices)
```

a|

> * The app provider needs to demonstrate that he is able to replace the RouterDevice with a new one and that the communication via this new RouterDevice can be continued.

Reviewer: ```suggestion
* The app provider has to demonstrate that he is able to replace the router device with a new one and that the communication via this new router device can be continued.
```
